README.md

```diff
@@ -73,6 +73,10 @@ to bring up a API documentation website (no interactive components).
 
 See the [documentation](https://brightlayer-ui-components.github.io/react-native) for information on using these components.
 
+### Upgrading to version 8
+
```

Comment:
should we add something here about version 8 in regards to the updated components, material design 3 and our updated theme

Reply:
maybe something like this but need some word smithing....

### Upgrading to version 8

Version 8 of [@brightlayer-ui/react-native-components](https://www.npmjs.com/package/@brightlayer-ui/react-native-components) uses React Native Paper v5 with Material Design v3 and is a major update with several changes. Most particularly, the Typography and IconWrapper components have been removed and we added 5 new components. In previous versions we supported Brightlayer-UI React Native Paper Themed Components and they have been removed.